ImperativeNavigation/Modules/Coordinator.swift

```diff
@@ -42,6 +43,10 @@ final class DefaultCoordinator: RoutableCoordinator {
 }
 
 extension DefaultCoordinator: Coordinator {
+    func popLast() {
+        path.remove(at: path.count - 1)
```

Comment:
```suggestion
        path.poplast()
```
This method will trigger a runtime exception if the path is empty. Also, according to docs, the complexity of [remove(at:)](https://developer.apple.com/documentation/swift/array/remove(at:)-1p2pj) is O(n). Instead, you can use [poplast()](https://developer.apple.com/documentation/swift/array/poplast()) which removes the last item if the collection is not empty, also the time complexity is O(1).